src/statements/WriteStatement.py

```diff
@@ -13,4 +13,5 @@ def __init__(self, variables, ctx: Wuwuzela_GrammarParser.WriteContext):
         comp = Composition(variables, ctx.compositionStatement())
 
         s = String(ctx.STRING())
-        write_Track("example6.mid", comp.nc)
+        s = s.value.replace('"', '')
```

Comment:
W sumie jak tak przeszkadza to w stringu, to można dać w konstruktorze `String` tą zmiankę

Reply:
Ale w sumie tylko tu używamy stringów, więc dużo nie zmieni

Reply:
wtedy z printowaniem chyba coś było nie tego
niech będzie tak, nie przeszkadza to tu